website/addons/base/__init__.py

```diff
@@ -350,6 +351,46 @@ def deep_url(self):
     def revision(self):
         return getattr(self, '_revision', None)
 
+    def add_tag(self, tag, auth, node, file_name, save=True):
```

Comment:
Why do we need these arguments?

* `file_name` can be accessed at `self.name` (I think...)
* `node` is accessible at `self.node`